packages/material-renderers/src/complex/TableToolbar.tsx

```diff
@@ -37,6 +37,7 @@ import { Grid, Typography } from '@mui/material';
 import AddIcon from '@mui/icons-material/Add';
 import ValidationIcon from './ValidationIcon';
 import NoBorderTableCell from './NoBorderTableCell';
+import { ArrayTranslations } from '@jsonforms/core/lib/i18n/arrayTranslations';
```

Comment:
This should be imported directly from the jsonforms/core package. If that is not possible, please export the new definitions appropriately.
```suggestion
import { ArrayTranslations } from '@jsonforms/core';
```